README.md

```diff
@@ -10,7 +10,7 @@
 ![Files](Files/Assets/FilesHome.png)
 
 ## What is Files?
-Files is a file manager which leverages the latest features of the Windows platform including Fluent Design, seamless updates, and APIs which enable the performance and lifecycle behavior that users expect. Whether you want to simplify your experience with your files or try something new, Files is a one-stop solution for exploring your files on the fly.
+Files is a file manager with a powerful yet intuitive design. It has features like multiple tabs, panes, columns, shell extensions in the context menu and tags.
```

Comment:
Packed with features like...

Reply:
I'm not so sure about the above features and how it is spelled

Reply:
Also, I liked the last sentence of the old text, could you squish it somewhere?

> Whether you want to simplify your experience with your files or try something new, Files is a one-stop solution for exploring your files on the fly.

Reply:
The text has already been decided and we've been using it in the store page, this PR is just to match the changes.